ProjectLighthouse/Helpers/CryptoHelper.cs

```diff
@@ -19,8 +19,8 @@ public static class CryptoHelper
     public static string GenerateAuthToken()
     {
         byte[] bytes = (byte[])GenerateRandomBytes(256);
-
-        return BCryptHash(Sha256Hash(bytes));
+        
```

Comment:
```suggestion

```
![image](https://github.com/LBPUnion/ProjectLighthouse/assets/16675503/b6438bf5-a4e9-4ef4-9e74-c89e3ce8fd1b)
me when i see extra whitespace in a pr

Reply:
extra whitespace, bruh I meant to put that there :|

Reply:
fine, but I'm not happy about it